todo-frontend-angular/src/app/todo/add-todo-modal/add-todo-modal.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, EventEmitter, Output, } from '@angular/core';
+import { FormControl,Validators, FormGroup } from '@angular/forms';
+import { NewTodoDTO } from 'src/app/models/new-todo-DTO.model';
+
+
+
+@Component({
+  selector: 'app-add-todo-modal',
+  templateUrl: './add-todo-modal.component.html',
+  styleUrls: ['./add-todo-modal.component.scss']
+})
+export class AddTodoModalComponent {
+  isVisible = false;
+  isOkLoading = false;
+  validateForm: FormGroup<{
+    title: FormControl<string | null>;
+    datePicker: FormControl<Date | null>;
+}> = new FormGroup({
+  title: new FormControl<string | null>(null, [Validators.required, Validators.minLength(1)]),
+  datePicker: new FormControl<Date | null>(null)
+}) 
+constructor(){}
+
+  @Output()
+  showModal(): void {
```

Comment:
I donøt think this needs to be an output :) Outputs are only when you want to communicate data to the parrent component